dev/setup-rancher-clusters

```diff
@@ -2,6 +2,12 @@
 
 set -eux
 
+# Check availability of Rancher CLI
+if ! command -v rancher &>/dev/null && ! command -v rancher-cli &>/dev/null; then
```

Comment:
This check is already run by `.github/scripts/register-downstream-clusters.sh`, called from L64; is the idea here to fail faster if the command is not available?